functions/src/scheduler/schedule.ts

```diff
@@ -23,3 +25,28 @@ export const deletePingsAfter30days: any = functions.pubsub.schedule('00 00 * *
     });
     return Promise.all(promises);
   });
+
+export const runAllMonitors60Mins: any = functions.pubsub.schedule('every 60 mins')
+  .onRun(async () => {
+    const snapshots: QuerySnapshot = await FirebaseAdmin.firestore()
+      .collection('projects')
+      .get();
+    
+    snapshots.docs.forEach(async (doc: firestore.QueryDocumentSnapshot) => {
+      const project: ProjectModel = <ProjectModel>doc.data();
```

Comment:
I dont think we want to redefine the `const` on each loop

Reply:
done